tests/e2e/scenarios/lib/common.sh

```diff
@@ -44,7 +44,6 @@ export KOPS_BASE_URL
 export KOPS
 export CHANNELS
 
-export KOPS_FEATURE_FLAGS="SpecOverrideFlag"
```

Comment:
I think the upgrade-ab scenario might need this.

Reply:
Should we set this specifically in those tests?

Reply:
That would be best.

Reply:
@hakman This environment variable needs to be added to the upgrade-ab scenario.

Reply:
Done. :)